src/Microsoft.Identity.Client/IClientApplicationBase.cs

```diff
@@ -75,7 +75,8 @@ public partial interface IClientApplicationBase
         /// <remarks>This is especially important when you deploy an application that you have initially tested locally; 
         /// you then need to add the reply URL of the deployed application in the application registration portal.
         /// </remarks>
-        string RedirectUri { get; set; }
+        string RedirectUri { get; } // TODO: note break in public API since this shouldn't be set after construction of object
```

Comment:
>TODO: note break in public API since this shouldn't be set after construction of obje [](start = 39, length = 85)

these seems reasonable : @jmprieur will surely be interested in reviewing :)